stacks-core/create-source-binary/build-scripts/Dockerfile.linux-musl-x64

```
@@ -7,6 +7,7 @@ ARG BUILD_DIR=/build
 ARG TARGET=x86_64-unknown-linux-musl
 ARG TARGET_CPU
 ENV RUSTFLAGS="${TARGET_CPU:+${RUSTFLAGS} -Ctarget-cpu=${TARGET_CPU}}"
+ARG TAG="No Tag Info"
```

```suggestion
ARG TAG
```
this should be enough - specifying a string for the tag isn't necessary if it's not set (i.e. the case will check for `singer-*` only)